src/components/review/ReviewCard.jsx

```
@@ -38,6 +42,27 @@ const ReviewCard = ({ review }) => {
     }
   }, [inView, scrollDirection, hasAnimated]);
 
+  // m3u8 재생을 위한 video type 설정 및 HLS.js 사용
+  useEffect(() => {
+    if (!videoRef.current || !file?.some((f) => f.fileType === 'VIDEO')) return;
```

ESLint

[error] 'review.file.some' is missing in props validation

(react/prop-types)